### PR TITLE
Misc. small improvements to README

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Various minor tweaks to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,3 +186,7 @@ cd packages/contracts
 pip3 install slither-analyzer
 yarn test:slither
 ```
+
+## License
+
+Files within this repository are licensed under the [MIT License](https://github.com/ethereum-optimism/optimism/blob/master/LICENSE) unless stated otherwise.
```
Comment (tynes) [inline README.md:192]: This is not true, `l2geth` is GNU GPLv3 - https://gist.github.com/kn9ts/cbe95340d29fc1aaeaa5dd5c059d2e60
We should be sure to mention that as people who have broken the geth license in the past have caused controversy

Comment (smartcontracts) [reply README.md:192]: Will make a note of this

Comment (smartcontracts) [reply README.md]: Updated

**Review by karlfloersch — APPROVED** [review]
LGTM!

**Pull request merged**